.github/workflows/push-build.yaml

```diff
@@ -37,16 +40,24 @@ jobs:
           aws-region: us-east-1
 
       - name: unshallow
-        run: git fetch --prune --unshallow --tags
+        run: git fetch --prune --unshallow --tags --force
 
       # creating custom env var
       - name: set env
-        run: echo "BRANCH_NAME=${GITHUB_REF##*/}" >> $GITHUB_ENV
+        run: |
+          echo "BRANCH_NAME=${GITHUB_REF##*/}" >> $GITHUB_ENV
+          echo "GITHUB_REF"=${GITHUB_REF} >> $GITHUB_ENV
```

Comment:
What is the redirection for?

Reply:
we can get the exact branch name from `${GITHUB_REF##*/}` & to know branch or tags`${GITHUB_REF}` and to address this  [comment](https://github.com/rook/rook/pull/8328#discussion_r672388978)